src/System.Runtime/tests/System/Reflection/AssemblyTests.cs

```diff
@@ -257,55 +265,99 @@ public static void GetManifestResourceStream()
         public static void Test_GlobalAssemblyCache()
         {
             Assert.False(typeof(AssemblyTests).Assembly.GlobalAssemblyCache);
-        }        
+        }
 
         [Fact]
         public static void Test_HostContext()
         {
             Assert.Equal(0, typeof(AssemblyTests).Assembly.HostContext);
-        }        
+        }
 
         [Fact]
         public static void Test_IsFullyTrusted()
         {
             Assert.True(typeof(AssemblyTests).Assembly.IsFullyTrusted);
-        }        
+        }
 
         [Fact]
-        public static void Test_SecurityRuleSet()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "The full .NET framework supports SecurityRuleSet")]
+        public static void Test_SecurityRuleSet_Netcore()
         {
             Assert.Equal(SecurityRuleSet.None, typeof(AssemblyTests).Assembly.SecurityRuleSet);
-        }        
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "SecurityRuleSet is ignored in .NET core")]
+        public static void Test_SecurityRuleSet_Netfx()
+        {
+            Assert.Equal(SecurityRuleSet.Level2, typeof(AssemblyTests).Assembly.SecurityRuleSet);
+        }
 
         [Fact]
         public static void Test_LoadFile()
         {
             var assem = typeof(AssemblyTests).Assembly;
             string path = "System.Runtime.Tests.dll";
             string fullpath = Path.GetFullPath(path);
-            Assert.Throws<ArgumentNullException>("path", () => Assembly.LoadFile(null));
-            Assert.Throws<ArgumentException>(() => Assembly.LoadFile(path));
             var loadfile1 = Assembly.LoadFile(fullpath);
-            Assert.NotEqual(assem, loadfile1);
+            Assert.Equal(assem.FullName, loadfile1.FullName);
             string dir = Path.GetDirectoryName(fullpath);
             fullpath = Path.Combine(dir, ".", path);
             var loadfile2 = Assembly.LoadFile(fullpath);
-            Assert.Equal(loadfile1,loadfile2);
```

Comment:
Hmm, I get the following test failures: `msbuild /T:BuildAndTest /P:TargetGroup=netfx`

```
      System.Reflection.Tests.AssemblyTests.Test_LoadFile [FAIL]
        Assert.Equal() Failure
        Expected: System.Runtime.Tests, Version=4.2.0.0, Culture=neutral, PublicKeyToken=9d77cc7ad39b68eb
        Actual:   System.Runtime.Tests, Version=4.2.0.0, Culture=neutral, PublicKeyToken=9d77cc7ad39b68eb
        Stack Trace:
          C:\Users\hughb\Documents\GitHub\corefx\src\System.Runtime\tests\System\Reflection\AssemblyTests.cs(307,0): at
   System.Reflection.Tests.AssemblyTests.Test_LoadFile()
``